src/graphql/types/object/business-account.ts

```diff
@@ -37,10 +40,25 @@ const BusinessAccount = new GT.Object({
     csvTransactions: {
       type: GT.NonNull(GT.String),
       args: {
-        walletIds: {
-          type: GT.NonNullList(GT.ID),
+        walletNames: {
+          type: GT.NonNullList(WalletName),
         },
       },
+      resolve: async (source, args) => {
+        const walletIds: WalletId[] = []
+
+        for (const id of source.walletIds) {
```

Comment:
This should be implemented as a use case method under `app` _not_ directly in the resolver!

Reply:
Done